model_compression_toolkit/exporter/fully_quantized/keras/builder/fully_quantized_model_builder.py

```diff
@@ -96,11 +102,10 @@ def build_model(self) -> Tuple[Model, UserInformation]:
 
         def _wrap_layer_with_quantize_config(layer):
 
-            nodes = self.graph.find_node_by_name(get_node_name_from_layer(layer))
+            node = self.oh.layer_to_node_dict.get(layer)
 
-            if len(nodes) == 1:
-                node = nodes[0]
-                return QuantizeWrapper(layer, get_quantization_config(node))
+            if node is not None:
+                return FullyQuantizedQuantizeWrapper(layer, get_quantization_config(node), layer.output_shape)
```

Comment:
Add TODO to try to use this Wrapper in mixed-precision builder to allow using TFOpLambda in MP.